update_data.py

```diff
@@ -198,9 +199,16 @@ def insert_or_update(sql_args, cursor, table):
                               original_interested_pwgs = ?,
                               notes = ? WHERE uid = ?''' % (table), sql_args)
     else:
-        print('Inserting %s' % (nice_description))
-        cursor.execute('''INSERT INTO %s
-                          VALUES (NULL, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, 0)''' % (table), sql_args)
+        if(table != 'twiki_samples'):
+            print('Inserting %s' % (nice_description))
+            cursor.execute('''INSERT INTO %s
+                           VALUES (NULL, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, 0)''' % (table), sql_args)
+        else:
+            sql_args.append(target_num_events)
+            print('Inserting %s' % (nice_description))
+            cursor.execute('''INSERT INTO %s
```

Comment:
This takes care of adding new entry to `twiki_samples` table. What happens if it already exists in `twiki_samples` table? target_num_events will not be updated. Will it never change?